modules/tinymce/src/core/main/ts/api/Editor.ts

```diff
@@ -385,7 +385,10 @@ class Editor implements EditorObservable {
   /**
    * Executes a legacy callback. This method is useful to call old 2.x option callbacks.
    * There new event model is a better way to add callback so this method might be removed in the future.
+   * <br>
+   * <em>`execCallback` has been deprecated and marked for removal in TinyMCE 6.0.</em>
```

Comment:
```suggestion
   * <em><code>execCallback</code> has been deprecated and marked for removal in TinyMCE 6.0.</em>
```
Back ticks won't work here, so we need to use `code` instead.